hoomd/mpcd/stream.py

```diff
@@ -572,39 +572,46 @@ class sphere(_streaming_method):
     r""" Spherical streaming geometry.
```

Comment:
These changes probably should not be implemented here. This is the "standard" sphere geometry, which can use the normal confined streaming method. I would suggest that instead of making these changes, you force `V=0` when you make the geometry.

Then, we will want a new class that is a drying sphere. We can discuss how to implement that and some of the other things you were struggling with below.